mappings/i2b2/lab-results-mapping.json

```diff
@@ -0,0 +1,82 @@
+{
+  "id" : "lab-results-mapping",
+  "url" : "https://datatools4heart.eu/fhir/mappings/i2b2/lab-results-mapping",
+  "name" : "lab-results-mapping",
+  "title" : "Mapping from i2b2 observation fact to the DT4H LabResult profile",
+  "source" : [ {
+    "alias" : "observationFact",
+    "url" : "https://datatools4heart.eu/fhir/schemas/i2b2/ObservationFact",
+    "joinOn" : [ ]
+  } ],
+  "context" : {
+    "labResultCodes" : {
+      "category" : "concept-map",
+      "url" : "$CONTEXT_REPO/i2b2/lab-result-codes.csv"
+    }
+  },
+  "variable" : [ {
+    "name": "isLabResult",
+    "language": "text/fhirpath",
+    "expression": "CONCEPT_CD.startsWith('LOINC')"
+  }, {
+    "name": "labCodeValue",
+    "language": "text/fhirpath",
+    "expression": "CONCEPT_CD.split(':').last()"
+  }
+  ],
+  "mapping" : [ {
+    "expression" : {
+      "name" : "result",
+      "language" : "application/fhir-template+json",
+      "value" : {
+        "status" : "final",
+        "category" : [ {
+          "coding" : [ {
+            "system" : "http://terminology.hl7.org/CodeSystem/observation-category",
+            "code" : "laboratory",
+            "display" : "Laboratory"
+          } ]
+        } ],
+        "code" : {
+          "coding" : [ {
+            "system" : "http://loinc.org",
+            "code" : "{{%labCodeValue}}",
```

Comment:
Some codes does not exists in our valueset: https://github.com/DataTools4Heart/common-data-model/blob/29b62f72230c66bd48dd153d8ffaa4ba7ea37ee2/valuesets/lab-result.json

E.g.
![image](https://github.com/user-attachments/assets/5eea931e-f4b9-4ac5-9deb-14eeb5fa3e8b)
Can you check those labs if it is possible to convert or perform unit conversion or discard them?